tests/bluetooth/ctrl_isoal/src/isoal_test_common.h

```diff
@@ -37,9 +37,9 @@
 		(s == BT_ISO_CONT ? "CONT" : \
 			(s == BT_ISO_END ? "END" : "???"))))
 
-#define ROLE_TO_STR(s) (s == BT_ROLE_BROADCAST ? "Broadcast" : \
-	(role == BT_CONN_ROLE_PERIPHERAL ? "Peripheral" : \
-	(role == BT_CONN_ROLE_CENTRAL ? "Central" : "Undefined")))
+#define ROLE_TO_STR(s) (s == ISOAL_ROLE_BROADCAST_SINK ? "Broadcast Sink" : \
+	(role == ISOAL_ROLE_PERIPHERAL ? "Peripheral" : \
+	(role == ISOAL_ROLE_CENTRAL ? "Central" : "Undefined")))
```

Comment:
Since there is now a `ISOAL_ROLE_BROADCAST_SOURCE` role, I think this macro should also be expanded rather than ending up Undefined.

Reply:
Yes it is a bug, thanks to copy and paste :-). Sorry about that.
It should be s that is compared. But works right now because everywhere the macro is used,role is defined and passed into it.

Reply:
Fixed